apps/fhir/server/tests/test_authentication.py

```diff
@@ -79,6 +80,7 @@ def test_match_fhir_id_success(self):
         '''
         with HTTMock(self.fhir_match_hicn_success_mock, self.fhir_match_mbi_success_mock):
             fhir_id, hash_lookup_type = match_fhir_id(
+                mbi=self.test_mbi_hash,
```

Comment:
These necessary test updates may still leave a coverage hole. We may need a new test to show that we're actually searching using the mbis. Maybe one that creates something new to look up with a unique mbi and uses that mbi to search.